test/shadowed-ripple.html

```diff
@@ -0,0 +1,270 @@
+<!DOCTYPE html>
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Ripple in Shadow conditions</title>
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../../web-component-tester/browser.js"></script>
+  <script src="../../iron-test-helpers/mock-interactions.js"></script>
+
+  <link rel="import" href="../../polymer/polymer.html">
+  <link rel="import" href="../../iron-behaviors/iron-button-state.html">
+  <link rel="import" href="../paper-ripple-behavior.html">
+</head>
+<body>
+  <style>
+    #source {
+      height: 25px;
+      width: 50px;
+      background: green;
+    }
+  </style>
+
+  <dom-module id="sd-ripple">
```

Comment:
How about I move the tests into `paper-ripple-behavior.html` and leave the element definition in this file?